packages/lib/har/harUtils.ts

```diff
@@ -87,19 +95,45 @@ export const fromHarContent = (content?: HarFormatContent) => {
   if (content?.text) {
     return Buffer.from(content.text, content.encoding === 'base64' ? 'base64' : 'binary');
   }
+  if (content?.json) {
+    return content?.json;
+  }
   return Buffer.alloc(0);
 };
 
+const checkMimeTypeListAndParseJson = (
+  body: string | Buffer,
+  mimeType?: string,
+): HarFormatPostData => {
+  try {
+    return {
+      mimeType: mimeType,
```

Comment:
```suggestion
      mimeType,
```
(shorter way to mean the same thing)